packages/e2e-tests/transfer-tokens/models/SendMoney.js

```diff
@@ -0,0 +1,44 @@
+class SendMoneyPage {
+    constructor(page) {
+        this.page = page;
+    }
+    async navigate() {
+        await this.page.goto(`/send-money`);
+    }
+    async typeAmount(amount) {
+        await this.page.fill(
+            `data-test-id=sendMoneyAmountInput`,
+            (amount && amount.toString()) || ""
```

Comment:
Feels like this method should _only_ be responsible for typing a specific amount with an assumption that it should only ever be called with a valid value for `amount` -- I don't like the implicit fallback logic here that will produce an empty string if `amount` is falsy; this could hide bugs in our code, and if we need to be able to reset the content in a field I think it is safest to implement an explicit behavioural method to do that (e.g. 'resetAmount()`)